### ISSUE TITLE
Plasma 6 support

### ISSUE THREAD

**Issue by Dan-Sun** [dragotin/kraft#233]
Hey

Since plasma 6 is available kraft doesn't connect to akonadi anymore. So no contacts will be listed. Not a big issue but maybe something related to plasma 6 or just on my side?

I saw that the copyright is 2004. 20 years of Kraft! Unbelievable! Thanks for such a loooong time, still being there for it.

**Comment by dragotin** [comment]
Yes, I will look into porting to Qt6 soonish

**Comment by dragotin** [comment]
Also Port Grantlee -> [KTextTemplate](https://api.kde.org/frameworks/ktexttemplate/html/index.html)

**Comment by dragotin** [comment]
No, not instead of weasyprint, but instead of grantlee, which is the current template engine. Using KTextTemplate instead will happen with the porting to qt6/KF6.

The logfile above shows a different problem: Obviously your kraft version was linked with an incompatible version of Akonadi - probably the latest Akonadi version. That will also only be fixed with kraft going to KF6. So, please stay tuned.

**Comment by hieu-van** [comment]
This problem has also creeped into KDE PIM on Debian sid

**Comment by dragotin** [comment]
Kraft 2.0 is now based on Qt6, to be released soon.
